build/packages/rpm/server/SPECS/pmm-update.spec

```diff
@@ -5,20 +5,20 @@
 %define _binaries_in_noarch_packages_terminate_build   0
 %define _unpackaged_files_terminate_build   0
 
-%global repo            pmm-update
+%global repo            pmm
 %global provider        github.com/percona/%{repo}
 %global import_path     %{provider}
 %global commit	        592eddf656bce32a11bd958af0a32c62bd5ea34c
 %global shortcommit	    %(c=%{commit}; echo ${c:0:7})
 %define build_timestamp %(date -u +"%y%m%d%H%M")
-%define release         66
+%define release         67
 %define rpm_release     %{release}.%{build_timestamp}.%{shortcommit}%{?dist}
 
 # the line below is sed'ed by build/bin/build-server-rpm to set a correct version
 %define full_pmm_version 2.0.0
 
-Name:		%{repo}
-Version:	%{version}
+Name:		pmm-update
+Version:	%{full_pmm_version}
```

Comment:
I'm not sure why we are changing this value

Reply:
Without this change we're getting `Too many levels of recursion in macro expansion. It is likely caused by recursive macro declaration` error during rpm build.
Not sure how it worked before but I could not make it work without this change for `vmproxy` either.